docs/tutorial/install.rst

```diff
@@ -38,3 +38,15 @@ Install Microceph
     for i in $(seq 1 3); do
         lxc exec microceph-$i -- sh -c 'snap install microceph'
     done
+
+Update management
+~~~~~~~~~~~~~~~~~
+
+For the time being, it's recommended that updates be postponed indefinitely.
```

Comment:
This isn't a 'for the time being' thing, it's a 'this is how we recommend you install and manage the lifecycle of microceph. Fundamentally, we're never going to be able to manage microceph updates from within microceph so we have to recommend the correct workflows for our users to manage it themselves and that includes holding the version, as well as how to update when they're ready!